core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -271,6 +271,18 @@ private boolean isInternalTopic(final String topicName) {
             && (topicName.endsWith("-changelog") || topicName.endsWith("-repartition"));
     }
 
+    private void printHelp(OptionParser parser) throws IOException {
+        System.err.println("The Application Reset Tool allows you to quickly reset an application in order to reprocess its data from scratch.\n" +
```

Comment:
Can you please format this differently. No line should be longer than 120 chars and please start a new line after each sentence (otherwise reviewing is quite cumbersome). Thx